java/src/main/java/ai/rapids/cudf/ColumnView.java

```diff
@@ -674,11 +674,10 @@ public final ColumnVector[] slice(int... indices) {
       }
     } catch (Throwable t) {
       try {
-        cleanupColumnViews(nativeHandles, columnVectors);
-      } catch (Throwable s) {
-        t.addSuppressed(s);
+        cleanupColumnViews(nativeHandles, columnVectors, t);
+      } finally {
+        throw t;
```

Comment:
This should still have the catch instead of finally, otherwise if `cleanupColumnViews` throws (it shouldn't, but if it somehow did) then this will smash any exception thrown from `cleanupColumnViews`.

Comment applies to the other places this was done.